package.json

```diff
@@ -12,17 +12,16 @@
     "@daily-co/config-plugin-rn-daily-js": "0.0.4",
```

Comment:
Need to update to use 0.0.5 once we publish it.
Waiting for this [PR](https://github.com/daily-co/rn-daily-js-expo-config-plugin/pull/9).

Reply:
Also we should update to use the latest version of `react-native-webrtc: 118.0.3-daily.2`

Reply:
Done.